src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -93,6 +93,7 @@ public function getConfigTreeBuilder()
                 ->booleanNode('enable_entrypoint')->defaultTrue()->info('Enable the entrypoint')->end()
                 ->booleanNode('enable_docs')->defaultTrue()->info('Enable the docs')->end()
                 ->booleanNode('enable_profiler')->defaultTrue()->info('Enable the data collector and the WebProfilerBundle integration.')->end()
+                ->booleanNode('enable_workflow')->defaultTrue()->info('Enable the Workflow integration.')->end()
```

Comment:
It should default to the value of `interface_exists(WorkflowInterface::class)`.
May I suggest: 'Enable the integration with the Symfony Workflow component.'?